doc/source/serve/managing-deployments.md

```diff
@@ -87,7 +87,7 @@ and leave us feedback through [Github Issues](https://github.com/ray-project/ray
 
```

Comment:
Could you also remove "experimental" from the above?  If it's  still "beta", we can leave the warning about it being subject to change, as long as we remove the word "experimental"